### PR TITLE
Update to nx 10

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Update : 
- nx
- angular / material
- akita
- firebase
- @angular/fire

Delete :
- Spectator

### CODE REVIEW

File: libs/admin/package.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+{
+  "name": "@blockframes/admin",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
+    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+  }
,
```
Comment (phpgeekfr) [inline libs/admin/package.json:7]: linter

File: libs/contract/package.json [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^8.2.13",
-    "@angular/core": "^8.2.13"
+{
+  "name": "@blockframes/contract",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^8.2.13",
+    "@angular/core": "^8.2.13"
+  }
,
```
Comment (phpgeekfr) [inline libs/contract/package.json:7]: linter

File: libs/ethers/package.json [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+{
+  "name": "@blockframes/ethers",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
+    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+  }
,
```
Comment (phpgeekfr) [inline libs/ethers/package.json:7]: linter

File: libs/financial-report/package.json [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^8.2.13",
-    "@angular/core": "^8.2.13"
+{
+  "name": "@blockframes/financial-report",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^8.2.13",
+    "@angular/core": "^8.2.13"
+  }
,
```
Comment (phpgeekfr) [inline libs/financial-report/package.json:7]: linter

File: libs/movie/package.json [hunk 5]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+{
+  "name": "@blockframes/movie",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
+    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+  }
,
```
Comment (phpgeekfr) [inline libs/movie/package.json:7]: linter

File: libs/notification/package.json [hunk 6]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+{
+  "name": "@blockframes/notification",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
+    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+  }
,
```
Comment (phpgeekfr) [inline libs/notification/package.json:7]: linter

File: libs/organization/package.json [hunk 7]
```diff
@@ -1,8 +1,11 @@
...
-    "@angular/common": "^7.2.0",
-    "@angular/core": "^7.2.0"
+{
+  "name": "@blockframes/organization",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^7.2.0",
+    "@angular/core": "^7.2.0"
+  }
,
```
Comment (phpgeekfr) [inline libs/organization/package.json:7]: linter

File: libs/utils/src/lib/yandex-metrica/yandex-metrica.module.ts [hunk 8]
```diff
@@ -3,7 +3,7 @@ import { NgModule, ModuleWithProviders } from '@angular/core';
 
 @NgModule()
 export class YandexMetricaModule {
-  public static forRoot(config: number): ModuleWithProviders {
+  public static forRoot(config: number): ModuleWithProviders<YandexMetricaModule> {
```
Comment (phpgeekfr) [inline libs/utils/src/lib/yandex-metrica/yandex-metrica.module.ts:6]: Is this also automated change ?

Comment (GrandSchtroumpf) [reply libs/utils/src/lib/yandex-metrica/yandex-metrica.module.ts:6]: yes

**Pull request merged**